### DISCUSSION TITLE
Problem with using target vector with 'extract_relevant_features'

### DISCUSSION THREAD

**Discussion by anspag** [blue-yonder/tsfresh#935] [Q&A]
Hello,

I am Tsfresh for time series anaysis. I am using data format:

![image](https://user-images.githubusercontent.com/102151084/159461964-67f1ba54-8060-4bfc-aadd-157d417ec841.png)

and target vector for **'extract_relevant_features'** as:

![image](https://user-images.githubusercontent.com/102151084/159462067-a6841040-5beb-48f8-9fd0-810bc749f381.png)

I have made Y factor in these two formats:

1. 
![image](https://user-images.githubusercontent.com/102151084/159462147-0f117b61-6ed9-4184-a67f-f0ec7d2c16c0.png) 

-----> **Error**: No match is matrix sizes

2. 
![image](https://user-images.githubusercontent.com/102151084/159462693-c7d05a79-dda3-4774-84de-8d3a72a87a5a.png)

-----> **Errror:** np.array not hasable

**Here (1,2,3,4 are ID's of my data like A, B, C, D in the shown data frame)**
But both give some errors. Am I missing something here. What should I do ? Please specifiy how to frame the data frame of y vector. 

Thanks in advance:
anshul.pagariya@rwth-aachen.de

**Comment by huberl** [comment]
I think the second solution works if you leave out the ids so `y=np.array([0, 0, 0, 1])`
